api/base/serializers.py

```diff
@@ -29,6 +29,59 @@ def _url_val(val, obj, serializer, **kwargs):
         return val
 
 
+class HyperlinkedIdentityFieldWithMeta(ser.HyperlinkedIdentityField):
+    # Returns a list with a url and an optional hash containing additional meta information
```

Comment:
Yes, thanks, I also need to update the docstring to reflect that it returns a dictionary with url and optional meta information/link_type